### PR TITLE
Makes pierced reality thingys actually scary.

### PR DESCRIPTION
**Pull request by Djiq**
## About The Pull Request

Adds a bunch of interaction for pierced realities to make their presence have bigger impact on the game.
Also fixes a bug with illusions spawning only on a single tile of an area.. oops

Trying to interact with them using TK will cause your head to literally explode in a very flashy manner.

Trying to interact with them using your hand has a 25% chance of dismembering the limb and qdeling it.

## Why It's Good For The Game

Pierced realities are underwhelming in their effect. Currently examining them gives you 10 brain damage, this expands this 'conditional' deadliness onto diffrent aspects of trying to interact with them.

## Changelog
:cl:
add: Otherwordly presence has taken more control over the holes in reality.
fix: Influences now actually spawn randomly on the station.

/:cl:

### CODE REVIEW

File: code/modules/antagonists/eldritch_cult/eldritch_effects.dm [hunk 1]
```diff
@@ -171,6 +171,62 @@
...
+/obj/effect/broken_illusion/attack_tk(mob/user)
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/human_user = user
+	if(IS_HERETIC(human_user))
+		to_chat(human_user,"<span class='boldwarning'>You know better than to tempt forces out of your control!</span>")
+	else
+		//a very elaborate way to suicide
+		to_chat(human_user,"<span class='userdanger'>Eldritch energy lashes out, piercing your fragile mind, tearing it to pieces!</span>")
+		var/obj/item/bodypart/head/head = locate() in human_user.bodyparts
```
Comment (ATH1909) [inline code/modules/antagonists/eldritch_cult/eldritch_effects.dm:223]: what if you don't have a head

**Comment by ATH1909** [comment]
proofread your PRs reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

File: code/modules/antagonists/eldritch_cult/eldritch_effects.dm (hunk 1 above)
Comment (EdgeLordExe) [reply code/modules/antagonists/eldritch_cult/eldritch_effects.dm:223]: aaaaaaaaaa forgot hard existed

**Comment by IndieanaJones** [comment]
Seems kind of boring.

**Comment by EdgeLordExe** [comment]
> Seems kind of boring and pointless. The temp thing is sort of neat I guess

everything in this is essentially flavour, but it is the little things that matter in setting up the tone.

**Comment by Iatots** [comment]
Gasses could be an idea too, BZ and miasma especially. Tho QM might come around and turn it into a monkey printer.

**Comment by Krysonism** [comment]
I don't like the heating and cooling as this will only lead to early shuttle calls before the round can develop.

**Comment by EdgeLordExe** [comment]
actually yeah i agree with kryson

**Comment by Iatots** [comment]
Eh, one degree at a time. It's gonna take some effort from RNJesus to make any real difference, and people are probably going to wall them off with r-windows. Worst case scenario, they rcd the floor away too. But hold on, are they a dense object? I realize now I never tried walking into one. That could have an interesting effect applied, would make it fun in the middle of a hallway.

**Comment by EdgeLordExe** [comment]
> Eh, one degree at a time. It's gonna take some effort from RNJesus to make any real difference, and people are probably going to wall them off with r-windows. Worst case scenario, they rcd the floor away too. But hold on, are they a dense object? I realize now I never tried walking into one. That could have an interesting effect applied, would make it fun in the middle of a hallway.

they aren't dense, they are basically objects that have 0 impact on your game... unless you try to interact with them.

**Comment by ExcessiveUseOfCobblestone** [comment]
why would you ever interact with them? is it just noob bait?

**Comment by EdgeLordExe** [comment]
> why would you ever interact with them? is it just noob bait?

they already cause damage if you examine them, this is mainly for flavour reasons. You know, sets the mood for a heretic round. You percieve things diffrently when you know something **might** be a danger. Instead of 'oh yeah that yeah uh that just floats there' it actually increases the paranoia if you know that something can **potentially** be a danger.

**Comment by ExcessiveUseOfCobblestone** [comment]
so yes

**Pull request merged**